pkg/meta/utils/str_set.go

```diff
@@ -102,6 +102,15 @@ func (set *StrSet) AppendSet(target StrSet) {
 	}
 }
 
+//ToArray returns an array of all items in the set.
+func (set *StrSet) ToArray() utils.StringArray {
+	arr := utils.StringArray{}
+	for k := range *set {
+		arr = append(arr, k)
+	}
+	return arr
+}
```

Comment:
The idea of this func is to return all the keys of the map/set as an array. Wouldn't it be nicer to just return the keys marked as existent (value = true)?  
Btw, testing this function would be nice and easy I believe